gamestonk_terminal/fundamental_analysis/av_view.py

```diff
@@ -517,3 +517,176 @@ def clean_fundamentals_df(df_fa: pd.DataFrame, num: int) -> pd.DataFrame:
     df_fa.columns.name = "Fiscal Date Ending"
 
     return df_fa
+
+
+def mscore(other_args: List[str], ticker: str):
+    """Mscore for given ticker
+
+    Parameters
+    ----------
+    other_args : List[str]
+        argparse other args
+    ticker : str
+        Fundamental analysis ticker symbol
+    """
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        prog="cash",
+        description="""
+            The Beneish model is a statistical model that uses financial ratios calculated with
+            accounting data of a specific company in order to check if it is likely (high
+            probability) that the reported earnings of the company have been manipulated.
+            [Source: Wikipedia]
+        """,
+    )
+
+    parser.add_argument(
+        "-s",
+        "--sub",
+        action="store_true",
+        default=False,
+        dest="sub",
+        help="Show sub stats.",
+    )
+
+    parser.add_argument(
+        "-d",
+        "--description",
+        action="store_true",
+        default=False,
+        dest="description",
+        help="Show descriptions for stats.",
+    )
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+
+        if not ns_parser:
+            return
+
+        fd = FundamentalData(key=cfg.API_KEY_ALPHAVANTAGE, output_format="pandas")
+        # pylint: disable=unbalanced-tuple-unpacking
+        # pylint: disable=no-member
+        df_cf, _ = fd.get_cash_flow_annual(symbol=ticker)
+        df_bs, _ = fd.get_balance_sheet_annual(symbol=ticker)
+        df_is, _ = fd.get_income_statement_annual(symbol=ticker)
+        df_cf = df_cf.set_index("fiscalDateEnding").iloc[:2]
+        df_bs = df_bs.set_index("fiscalDateEnding").iloc[:2]
+        df_is = df_is.set_index("fiscalDateEnding").iloc[:2]
+
+        ar = [int(x) for x in df_bs["currentNetReceivables"].to_list()]
```

Comment:
Instead of converting the array to a list, you could have gathered the array of values directly, like this: `df_bs["currentNetReceivables"].values`

Reply:
Alpha Vantage returns all values as strings and this was the simplest way I could think of to convert them to integers. Using the values method returns them as strings, but if there is an easier way to return as integers let me know.

Reply:
You can do df[column] = df[column].apply(lambda x: float(x))